From a4386054dbb96267cd5eeb23490cefb26f80dbc7 Mon Sep 17 00:00:00 2001
From: Alexander Amelkin <alexander@amelkin.msk.ru>
Date: Sun, 17 May 2020 00:34:03 +0300
Subject: [PATCH] Disable the floating poing features

With CONFIG_MP_VHT_HW_TX_MODE enabled the build wants
to use -mfloat-abi=hard which conflicts with the LibreELEC
-msoft-float. This patch disables the MP_VHT_HW_TX_MODE
to get rid of the floating point ABI dependency.

Signed-off-by: Alexander Amelkin <alexander@amelkin.msk.ru>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6a065bf..d7aad0b 100755
--- a/Makefile
+++ b/Makefile
@@ -99,7 +99,7 @@ CONFIG_AP_WOWLAN = n
 ######### Notify SDIO Host Keep Power During Syspend ##########
 CONFIG_RTW_SDIO_PM_KEEP_POWER = y
 ###################### MP HW TX MODE FOR VHT #######################
-CONFIG_MP_VHT_HW_TX_MODE = y
+CONFIG_MP_VHT_HW_TX_MODE = n
 ###################### Platform Related #######################
 CONFIG_PLATFORM_I386_PC = y
 CONFIG_PLATFORM_ARM_RPI = n
-- 
2.17.1

